packages/modular-scripts/src/utils/getPackageMetadata.ts

```diff
@@ -56,9 +56,10 @@ async function getPackageMetadata() {
   // validate tsconfig
   // Extract configuration from config file and parse JSON,
   // after removing comments. Just a fancier JSON.parse
+  const userTsConfigPath = path.join(modularRoot, typescriptConfigFilename);
```

Comment:
This is a bugfix. The second argument was previously reading `tsconfig.json` from the `cwd()`, since no modular root was being used. This is not consistent with the first argument, where the modular root is used.

That said, in practice, there was no actual way for users to be affected by this bug - since commands are supposed to be run in the root already (so not using the modular root resulted in the correct file being used), and preflight errors actually get in the way first if you try to run something like `typecheck` from the non-root dir.

The bugfix here is needed so that we can effectively test the code in test, i.e. use a faked modular root and have it respected.